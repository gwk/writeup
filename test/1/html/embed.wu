writeup v0

# txt
<embed: test/assets/text.txt>

# csv
## empty
<embed: test/assets/empty.csv>
## non-empty
<embed: test/assets/table.csv>

# svg
<embed: test/assets/circle.svg>

# img
<embed: test/assets/head.jpg> <embed: test/assets/head.gif> <embed: test/assets/head.png>
