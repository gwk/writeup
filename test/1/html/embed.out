<section class="S1" id="s0">
  <h1 id="h0">txt</h1>
  <p>
    <div class="code-block">
    <code class="line">Text contents.
</code>
    </div>
  </p>
</section>
<section class="S1" id="s1">
  <h1 id="h1">csv</h1>
  <section class="S2" id="s1.1">
    <h2 id="h1.1">empty</h2>
    <p>
      <table>
      </table>
    </p>
  </section>
  <section class="S2" id="s1.2">
    <h2 id="h1.2">non-empty</h2>
    <p>
      <table>
      <thead><tr>
        <th>A</th><th>B</th><th>C</th>
      </tr></thead><tbody>
        <tr><td>1</td><td>2</td><td>3</td></tr>
        <tr><td>4</td><td>5</td><td>6</td></tr>
      </tbody>
      </table>
    </p>
  </section>
</section>
<section class="S1" id="s2">
  <h1 id="h2">svg</h1>
  <p>
    <svg width="4em" height="4em" viewBox="0 0 4 4">
    Â  <circle cx="2" cy="2" r="1"/>
    </svg>
  </p>
</section>
<section class="S1" id="s3">
  <h1 id="h3">img</h1>
  <p>
    <img src=test/assets/head.jpg> <img src=test/assets/head.gif> <img src=test/assets/head.png>
  </p>
</section>
<section class="S1" id="s4">
  <h1 id="h4">wu</h1>
  <p>
    <p>
      line.
    </p>
  <br />
    <p>
      Relative:
    <br />
      <a href="line.wu">line.wu</a>
    <br />
      <p>
    line.
  </p>
    </p>
    <p>
      Absolute:
    <br />
      <a href="/test/assets/line.wu">/test/assets/line.wu</a>
    <br />
      <p>
    line.
  </p>
    </p>
  </p>
</section>
